development/compiling/compiling_for_windows.rst

```diff
@@ -69,6 +75,18 @@ environment variable after installing it, then check again.
 You can do so by running the Python installer again and enabling the option
 to add Python to the ``PATH``.
 
+.. note:: SCons installed via ``pip`` puts an executable ``scons.exe``
+          in the Scripts directory of your Python installation, or in a shadow script directory
+          if you did a User Install. For Windows to run ``scons`` as a command, you'll need this in your search path.
+          During installation with ``pip`` you might have missed a warning like this:
+          .. code-block:: pycon
+             WARNING: The scripts scons-configure-cache.exe, scons.exe and sconsign.exe
+             are installed in 'C:\Users\me\AppData\Roaming\Python\Python310\Scripts'
+             which is not on PATH.
+             Consider adding this directory to PATH or, if you prefer to suppress this warning,
+             use --no-warn-script-location.
+          
```

Comment:
Extra line here.

Reply:
sorry, which line number is extra?

Reply:
You have two empty lines there. I think one is enough.